pkg/gadgets/trace/tcpdrop/types/types.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022-2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/columns"
+	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
+)
+
+type Event struct {
+	eventtypes.Event
+	eventtypes.WithMountNsID
```

Comment:
Since this gadget doesn't provide filtering capabilities by container, I think we should remove this field as the mountns id is not actually related to the container that created the connection.